core/src/test/java/org/apache/accumulo/core/conf/PropertyTest.java

```diff
@@ -71,8 +71,8 @@ public void testProperties() {
           "Description not set for " + prop);
 
       // make sure property description ends with a period
-      assertTrue(prop.getDescription().endsWith("."),
-          "Property: " + prop.getKey() + " description does not end with period.");
+      assertTrue(prop.getDescription().trim().endsWith("."), "Property: " + prop.getKey()
+          + " description does not end with period. Description = " + prop.getDescription());
```

Comment:
Does this fail with the multi-line strings if we don't trim?

Reply:
Yes, it fails because of the newline at the end of the last line after the period added by the multi-line string feature.